src/GraphQL/packages.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Antlr4" version="4.5-alpha002" targetFramework="net45" userInstalled="true" />
```

Comment:
The alpha version is 1.5 years ahead of "stable".  I wanted to be on the latest antlr version.  The official GraphQL nuget il-merges the Antlr runtime so there should be no policy issue in that sense.  Not sure I want to make this change without metrics.